ext/manifest.json

```diff
@@ -67,5 +67,20 @@
   },
   "permissions": [
     "activeTab"
+  ],
+  "web_accessible_resources": [
```

Comment:
🔍 This is necessary to allow https://github.com/ZeroSharp/Chess.com_Analysis_Chrome_extension/pull/16/files#diff-2eef33af90ac05fc4e89de843350a3f3214e6d33f4a8686a3f547a075247fcedR51